### ISSUE TITLE
Several tests "Aborted" on CentOS

### ISSUE THREAD

**Issue by CIPop** [dotnet/corefx#8367]
From : http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/master/job/centos7.1_debug_tst_prtest/858/consoleFull#6262143651f1a4601-6aec-4fd5-b678-78d4389fd5e8

```
15:26:50 ./RunTests.sh: line 198: 33206 Aborted                 ./corerun xunit.console.netcore.exe System.Reflection.TypeExtensions.Tests.dll -xml testResults.xml -notrait Benchmark=true -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing -notrait category=nonlinuxtests
15:26:50 Finished running tests. End time=22:28:27. Return value was 134
15:26:50 error: One or more tests failed while running tests from ''.  Exit code 134.
15:26:50 Finished running tests. End time=22:28:27. Return value was 0
15:26:50 ./RunTests.sh: line 198: 33345 Segmentation fault      ./corerun xunit.console.netcore.exe System.Reflection.Tests.dll -xml testResults.xml -notrait Benchmark=true -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing -notrait category=nonlinuxtests
15:26:50 Finished running tests. End time=22:28:27. Return value was 139
15:26:50 error: One or more tests failed while running tests from ''.  Exit code 139.
15:27:06 ./RunTests.sh: line 201: 40586 Aborted                 ./corerun xunit.console.netcore.exe System.Runtime.Tests.dll -xml testResults.xml -notrait Benchmark=true -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing -notrait category=nonlinuxtests
15:27:06 Finished running tests. End time=22:28:43. Return value was 134
```

**Comment by ellismg** [comment]
@MattGal, have you seen this "aborted" thing before?

**Comment by stephentoub** [comment]
Looks like a seg fault.

**Comment by joshfree** [comment]
Could this be related to https://github.com/dotnet/corefx/issues/8501 ?

**Comment by ellismg** [comment]
It's probable.

**Comment by joshfree** [comment]
Moving to 1.1 until there's more info to go on.

**Comment by mellinoe** [comment]
There's been no more discussion of this (and the original log link 404's) so I'll assume the root bug was fixed as part of 1.0.0.
